### PR TITLE
card image, mana cost, spotter view fixes

### PR DESCRIPTION
**Pull request by kprevas**

### CODE REVIEW

File: gamestate.js [hunk 1]
```diff
@@ -10,25 +10,40 @@ modeRef.on('value', function (v) {
...
         // exact match on the name, because autocomplete in director.html should have
         // put an exact name there for us
         $.get("https://api.magicthegathering.io/v1/cards?name=\"" + cardName + "\"",
             function (data) {
                 if (data.cards && data.cards.length > 0) {
+                    data.cards = _.sortBy(data.cards, function (c) {
+                        return sets.indexOf(c.set);
+                    });
                     var card = data.cards[0];
+                    // for (var i = 0; i < data.cards.length; i++) {
```
Comment (cdanis) [inline gamestate.js]: ????

Comment (kprevas) [inline gamestate.js]: done

**Comment by kprevas** [comment]
did like 6 more things, PTAL

File: spotter.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 var spotterControlTemplate = _.template(`
```
Comment (cdanis) [inline spotter.js]: I didn't even realize underscorejs had this

**Pull request merged**